### ISSUE TITLE
Echo something to stdout when LS has successfully started

### ISSUE THREAD

**Issue by markwalkom** [elasticsearch/logstash#2163]
For the first time in ages today I called the Logstash binary directly, however it's not clear when it has actually started unless you use the -v or -vv flags.
eg I only got a response after I hit enter;
```bash
$ bin/logstash -f conf/ls.conf

2014-12-03T04:40:30.868+0000 bender.local
```

It'd be good to have something echoed out saying "Logstash now running" (or whatever) to let the user know it's started, similar to when it shuts down.

**Comment by purbon** [comment]
+1 on call it console.

**Comment by purbon** [comment]
Added the terminal method to be used here at https://github.com/jordansissel/ruby-cabin/pull/25, waiting for review and merge to then be integrated here.

**Comment by suyograo** [comment]
Closed by #2409
